app/main.ipynb

```diff
@@ -151,15 +262,78 @@
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
    "source": [
-    "# write your code here"
+    "\n",
+    "df[\"day\"] = df[\"timestamp\"].dt.day_name()\n",
+    "daily_income = df.groupby(\"day\")[\"quantity\"].sum().reset_index()\n",
+    "\n",
+    "sort_order = [\"Monday\", \"Tuesday\", \"Wednesday\", \"Thursday\", \"Friday\", \"Saturday\", \"Sunday\"]\n",
+    "daily_income = daily_income.sort_values(by=\"day\", key=lambda x: pd.Categorical(x, categories=sort_order))\n",
+    "\n",
+    "\n",
+    "plt.bar(daily_income.day, daily_income.quantity)\n",
+    "plt.xlabel(\"Order day\")\n",
+    "plt.ylabel(\"Total Sum of Order Prices\")\n",
+    "plt.title(\"Restaurant Income by Order Day\")\n",
+    "plt.xticks(range(7), daily_income.day)\n",
+    "\n",
+    "plt.show()"
    ],
    "metadata": {
     "collapsed": false,
     "pycharm": {
      "name": "#%%\n"
+    },
+    "ExecuteTime": {
+     "end_time": "2023-05-17T20:42:05.036731252Z",
+     "start_time": "2023-05-17T20:42:04.835999762Z"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 96,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Finally, the highest income during the week occurs on the weekends, especially on Friday and Saturday. However, the income levels are relatively average on working days.\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"Finally, the highest income during the week occurs on the weekends, especially on Friday and Saturday. However, the income levels are relatively average on working days.\")"
```

Comment:
You can add it as markdown. "Conclusions: ..." and your text or something like that to show that it is not a task description